README.md

```diff
@@ -1,6 +1,44 @@
 # Turtle
 
-**TODO: Add description**
+Turtle Graphics is a popular way to teach programming, particularly for kids. It
+was initially part of the Logo programming language, created by Wally Feurzing
+and Seymour Papert in 1966. Imagine a robotic turtle that starts in `{0,0}` on
+the cartesian plane. After importing the `Turtle` module, you would be able to
+use `IEx` (the interactive shell for Elixir) to send commands like `forward(15)`
+and get the turtle to advance 15 pixels forward, drawing a line while moving.
+You can draw pretty complex figures and
+pictures by combining that with other commands.
+
+This repository includes an experimental Elixir implementation about the
+previous ideas, it should allow users to do all that using the Scenic
+framework.
+
+## How to run the examples
+
+```console
+$ cd turtle_ui
+$ iex -S mix
+```
+
+And then, inside IEx you can do:
+
+```iex
+iex(1)> for file <- Path.wildcard("examples/*.ex"), do: Code.compile_file(file)
```

Comment:
Maybe an example with c "examples/sierpinski_triangle.ex" would make sense to include in documentation as well. This will make it easier for people to understand when creating their own programs in other folders.